### DISCUSSION TITLE
Language stat goes wrong on vercel deployment

### DISCUSSION THREAD

**Discussion by gfg7** [anuraghazra/github-readme-stats#2747] [Q&A] [answered]
Hi
I want to deploy my own instance of GitHub stats on vercel because of `maximum retries exceeded` thing
I changed [vercel.json file](https://github.com/gfg7/github-readme-stats/blob/399513a7ebab41c7220f1f7ba043314ba9ae11b4/vercel.json)
But what i see is language stats are wrong on my instance even though i use the _same_ url parameters
How to fix it? Thank you in advance

github-readme-stats.vercel.app
[![GitHub stats](https://github-readme-stats.vercel.app/api/top-langs/?username=gfg7&show_icons=true&count_private=false&layout=compact&bg_color=82,EF8B59,B53BC4&title_color=fff&text_color=fff&hide=css,scss,html)](https://github.com/gfg7/github-readme-stats)
[Raw link in case it doesnt't show](https://github-readme-stats.vercel.app/api/top-langs/?username=gfg7&show_icons=true&count_private=false&layout=compact&bg_color=82,EF8B59,B53BC4&title_color=fff&text_color=fff&hide=css,scss,html)

Mine vercel github-readme-stats-gfg7.vercel.app
[![GitHub stats](https://github-readme-stats-gfg7.vercel.app/api/top-langs/?username=gfg7&show_icons=true&count_private=false&layout=compact&bg_color=82,EF8B59,B53BC4&title_color=fff&text_color=fff&hide=css,scss,html)](https://github.com/gfg7/github-readme-stats)

**Comment by qwerty541** [comment, marked as answer]
Hey, Julia! The most likely reason for the different statistics on top languages card is different GitHub API tokens. Public vercel instance contain API token that cannot see your private repositories. Your own vercel instance with your own API token can see them.

**Reply by gfg7** [reply]
So that was the case. easy fix
Thank you!
